src/main/scala/sbtcatalysts/CatalystsPlugin.scala

```diff
@@ -253,7 +253,12 @@ trait CatalystsBase {
   )
 
   /** Combines all scalac options.*/
-  lazy val scalacAllOptions = scalacCommonOptions ++ scalacLanguageOptions ++ scalacStrictOptions ++ partialUnificatonSettings
+  lazy val scalacAllOptions = scalacCommonOptions ++ scalacLanguageOptions ++ scalacStrictOptions
```

Comment:
Let's add types to these declarations so we can't accidentally do it again.

Reply:
good point. added